src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableContentBlocks.tsx

```diff
@@ -274,6 +274,7 @@ const EditableContentBlock = ({
                   canDelete={!!canAddBlocks && !isReordering}
                   block={block}
                   id={id}
+                  releaseId={editingContext.releaseId}
```

Comment:
Instead of passing down the releaseId, (which I know will soon no longer exist on editingContext), can you use the ManageRelease context initialised at the root of the ManageReleasePage? and just use it where it's necessary at the bottom level, instead of passing it down?

https://github.com/dfe-analytical-services/explore-education-statistics/blob/master/src/explore-education-statistics-admin/src/pages/release/ManageReleasePageContainer.tsx

Reply:
Added `ManageReleaseContext` to `EditableDataBlock` and removed this.